manager/src/main/java/com/redhat/service/smartevents/manager/BridgesServiceImpl.java

```diff
@@ -94,8 +103,11 @@ public Bridge createBridge(String customerId, String organisationId, String owne
         bridge.setEndpoint(dnsService.buildBridgeEndpoint(bridge.getId(), customerId));
 
         //Ensure we connect the ErrorHandler Action to the ErrorHandler back-channel
-        Action errorHandler = bridgeRequest.getErrorHandler();
-        bridge.setDefinition(new BridgeDefinition(Objects.nonNull(errorHandler) ? errorHandler : null));
+        Action reqErrorHandler = bridgeRequest.getErrorHandler();
+        Action errorHandler = Objects.nonNull(reqErrorHandler) && ENDPOINT_ERROR_HANDLER_TYPE.equals(reqErrorHandler.getType())
```

Comment:
Rather than the constant for `ENDPOINT_ERROR_HANDLER_TYPE`, why not just `errorHandlerService.getDefaultErrorHandlerAction().getType()`

Reply:
This was refactored away. Anyway this was to avoid creating a new `Action` object only for checking the type.